src/auth_service/auth_adapter/api/main.py

```diff
@@ -55,21 +55,33 @@
     API_EXT_PATH += "/"
 
 
-def add_public_routes():
-    """Add all public (unprotected) routes."""
-    for route in CONFIG.public_paths:
-        if route.endswith("/*"):
-            route = route[:-1] + "{path:path}"
-        elif "*" in route:
-            route = route.replace("*", "{variable}")
-
-        @app.api_route(route, methods=READ_METHODS)
-        async def public_route() -> dict:
-            """Unprotected route."""
-            return {}
-
-
-add_public_routes()
+def add_allowed_route(route: str, write: bool = False):
+    """Add a route that shall be passed through."""
+    methods = WRITE_METHODS if write else READ_METHODS
```

Comment:
shouldn't it be rather:

```suggestion
    methods = ALL_METHODS if write else READ_METHODS
```

Reply:
No, this is intended. You can add the same path to read and write if you want both.

For instance, POST /work-packages is used by the browser, but GET /work-packages is used by the CLI.